.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -26,6 +26,10 @@ body:
       description: "Run `cargo semver-checks --bugreport` and copy-paste the output here."
     validations:
       required: true
+  - type: textarea
+    attributes:
+      label: "Build Configuration"
+      description: "Any local Rust build configuration in config.toml files? Paste the file's content here."
```

Comment:
Maybe "Paste the file's content here if you have one", to make it clear that not having one is normal (and possibly expected).